tests/unit/test_kernel.py

```diff
@@ -284,6 +285,99 @@ def expected_divergence_x_grad_y(
         """Compute expected divergence of the kernel w.r.t ``x`` gradient ``y``."""
 
 
+class TestLinearKernel(
+    BaseKernelTest[LinearKernel],
+    KernelRowSumTest[LinearKernel],
+    KernelGradientTest[LinearKernel],
+):
+    """Test ``coreax.kernel.LinearKernel``."""
+
+    @pytest.fixture
+    def kernel_factory(self) -> _KernelFactory[LinearKernel]:
+        def kernel(length_scale, output_scale):
+            return LinearKernel(length_scale, output_scale)
+
+        return kernel
+
+    @override
+    @pytest.fixture(
+        params=[
```

Comment:
Please remove the trailing comma from arrays. Ruff will then format this on one line.

Reply:
Fixed